From c0c605dd1fcb5afe8dfee3e7609f83a8b0e64820 Mon Sep 17 00:00:00 2001
From: ShadowKyogre <shadowkyogre.public@gmail.com>
Date: Fri, 24 Sep 2021 17:59:30 -0700
Subject: [PATCH] plugins/svg: Fix building against librsvg 2.51.0+

A commit introduced in librsvg 2.51.0+ renamed
librsvg/librsvg-features.h to librsvg/rsvg-features.h.

As a result, this broke the SVG plugin compile process.
The following patch includes a fix for this.

---

For the curious, here's the commit in librsvg that caused this.

https://github.com/GNOME/librsvg/commit/b8c8756a0e8148afb2752acb62f06c57c917e4b4
---
 plugins/svg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/svg.c b/plugins/svg.c
index a0eb715f..ee1b6446 100644
--- a/plugins/svg.c
+++ b/plugins/svg.c
@@ -32,7 +32,7 @@
 #include <librsvg/rsvg-cairo.h>
 #endif
 #ifndef LIBRSVG_FEATURES_H
-#include <librsvg/librsvg-features.h>
+#include <librsvg/rsvg-features.h>
 #endif
 
 #include <X11/Xatom.h>
-- 
GitLab

